src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{508A7D81-6462-459C-9F8F-B58FCCCFC8E7}</ProjectGuid>
+    <DefineConstants>$(DefineConstants);TESTING_CSP_IMPLEMENTATION</DefineConstants>
```

Comment:
@bartonjs This is for being able to tell if I use CSP vs CNG for skipping on desktop I'm using SkipOnTargetFramework. I'm not 100% sure if S.S.Cryptography.Algorithms.Tests implies CSP since this technically could be managed implementation too